sormas-backend/src/main/java/de/symeda/sormas/backend/info/EntityColumn.java

```diff
@@ -171,7 +172,29 @@ private static String getCaption(FieldData fieldData) {
 	}
 
 	private static String getDescription(FieldData fieldData) {
-		return I18nProperties.getPrefixDescription(fieldData.getI18NPrefix(), fieldData.getField().getName(), "");
+		String prefixDescription = I18nProperties.getPrefixDescription(fieldData.getI18NPrefix(), fieldData.getField().getName(), "");
```

Comment:
please rename to fieldDescription

Reply:
The data protection dictionary is country disease and feature dependent so it should not have this extra description, but should not show personal or sensitive in the data protection colum